docs/concepts.md

```diff
@@ -253,6 +248,12 @@ allowed (via two uni-directional Channels) if
 `(L_a ⊑ L_b) ∧ (L_b ⊑ L_a) ⇒ L_a = L_b`, i.e. if `a` and `b` have identical
 confidentiality and integrity.
 
+The least privileged label is usually referred to as "public untrusted" (and
+represented as `⊥`, pronounced "bottom"), which corresponds to a Node or Channel
+which has only observed public data, and its inputs are not endorsed with any
+level of integrity; in this label, both confidentiality and integrity components
+are empty sets.
```

Comment:
Moves this down, as to not interrupt the section of specific `L_a, L_b` examples.